Packs/RedLock/Integrations/RedLock/RedLock.py

```diff
@@ -359,6 +362,44 @@ def translate_severity(alert):
     return 0
 
 
+def get_remediation_details():
+    """
+    Retrieve remediation details for a given alert
+    """
+    alert_ids = argToList(demisto.getArg('alert-id'))
+    payload = {'alerts': alert_ids, 'filter': {}}
+    handle_filters(payload['filter'])
+    handle_time_filter(payload['filter'], {'type': 'to_now', 'value': 'epoch'})
+
+    if not alert_ids:
+        return_error('You must specify the alert-id to retrieve with CLI remediation')
+
+    response = req('POST', 'alert/remediation', payload, None)
+
+    if response:
+        for alert_id in alert_ids:
+            details = {
+                'ID': alert_id,
+                'Remediation': {
+                    'CLI': response['alertIdVsCliScript'][alert_id],
+                    'Description': response['cliDescription']
+                }
+            }
+            context = {
+                'Redlock.Alert(val.ID == obj.ID)': details
+            }
+            MD = tableToMarkdown("remediationCLIoutput", details)
```

Comment:
Don't pass all the details to tableToMarkdown, otherwise it will create not human friendly table. Instead create new variable with fields
```
human_readable_details = {
     'ID': details.get('ID'),   
     'RemediationCLI': details.get('Remediation').get('CLI'),
     'RemediationDescription': details.get('Remediation').get('Description')
}
```

Reply:
Added the recommended dictionary.  Any plans to support nested dictionaries in `tableToMarkdown` function?

Reply:
Unfortunately not in the near time